client/src/partials/patients/record/patient_record.js

```diff
@@ -1,26 +1,59 @@
 angular.module('bhima.controllers')
 .controller('PatientRecordController', PatientRecordController);
 
-PatientRecordController.$inject = ['$stateParams', 'PatientService', 'NotifyService', 'moment'];
+PatientRecordController.$inject = ['$stateParams', 'PatientService', 'NotifyService', 'moment', 'Upload', '$timeout', '$translate'];
 
-function PatientRecordController($stateParams, Patients, Notify, moment) {
+function PatientRecordController($stateParams, Patients, Notify, moment, Upload, $timeout, $translate) {
   var vm = this;
   var patientID = $stateParams.patientID;
   
   vm.loading = true;
+  vm.uploadFiles = uploadFiles;
+  vm.textButton = $translate.instant('PATIENT_RECORDS.UPLOAD_PICTURE');
+ 
+  refreshPatientsRead();
 
-  /** @fixme if no uuid is provided this will download all the patients through the base url '/' */
-  Patients.read(patientID)
-    .then(function (result) {
-      vm.patient = result;
-      vm.loading = false;
+  function uploadFiles(file, errFiles) {
```

Comment:
Just a quick note that if the file is valid it will be passed into `file`, if the file is invalid it will be passed into `errFiles`. 

This `if` structure ignores execution if there is no file, so if there is a problem with the image no error will be displayed. 

This structure could look something like:
```js
function uploadFiles(files, invalidFiles) { 
if (invalidFiles) {
  var firstError = invalidFiles[0].$error;

  // handle the error - it's worth noting that having an $error object like this 
  // is very similar to form validation - perhaps ng-messages could be used here
  return;
}

// no invalid files - upload files... 
```